groups/script.js

```diff
@@ -193,14 +193,23 @@ searchInput.addEventListener('keyup', () => {
   debounce(() => {
     const searchValue = searchInput.value.toUpperCase();
     const groupRoles = document.querySelectorAll('.group-role');
+    let foundResults = false;
     groupRoles.forEach((groupRole) => {
       const paragraphElement = groupRole.getElementsByTagName('p')[0];
       const paragraphContent = paragraphElement.textContent;
       const displayValue =
         paragraphContent.toUpperCase().indexOf(searchValue) > -1 ? '' : 'none';
       groupRole.style.display = displayValue;
+
+      if (displayValue === '') {
+        foundResults = true;
+      }
       loader.classList.add('hidden');
     });
+    const noResultsMessage = document.getElementById('no-results-message');
+    foundResults
+      ? (noResultsMessage.style.display = 'none')
+      : (noResultsMessage.style.display = 'block');
```

Comment:
Can we simplify this to something like this: 

```javascript
noResultsMessage.style.display = foundResults ? 'none' : 'block';
```

Reply:
Yes, we can do that. Thanks for pointing @Aryex82

Reply:
@Aryex82 I have done the changes. Please review the changes and approve.